pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -27168,15 +27168,15 @@ self: {
      }) {};
```

Comment:
You need to remove the changes to this file.

Reply:
Hi @mpickering thanks for the review.

When I originally sent this PR, I think I might have based it on the wrong commit, which is why it appeared that I had made changes to `pkgs/development/haskell-modules/hackage-packages.nix`.

I have fixed it in commit e0e2b3f.